latch_cli/constants.py

```diff
@@ -35,6 +35,7 @@ class LatchConstants:
 
     pkg_name: str = "latch"
     pkg_config: str = ".latch/config"
+    pkg_workflow_name: str = ".latch/workflow_name"
```

Comment:
can u put the wf_name in the config file instead

Reply:
I tried, but it is really mess with how the config is being used (expected to be created in some places, expected to be not created in some places). Going to put in a separate file.